bundles/core/org.openhab.core.library/src/main/java/org/openhab/core/library/types/PointType.java

```diff
@@ -113,6 +118,24 @@ public DecimalType getGravity() {
 	}
 
 	/**
+	 * Return the distance in meters from otherPoint, ignoring altitude. This algorithm also
```

Comment:
It's based on the [Haversine formula](https://en.wikipedia.org/wiki/Haversine_formula).